src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+
+  const MIDDLE_TERM = 3;
+  const MIDDLE_TERM_DISCOUNT = 20;
+
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  let price = days * RENT_COST;
+
+  if (days >= MIDDLE_TERM && days < LONG_TERM) {
+    price -= MIDDLE_TERM_DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
+    price -= LONG_TERM_DISCOUNT;
+  }
+
+  return price;
```

Comment:
Good job! You also can do like in this example:
```javascript
if (days >= MIDDLE_TERM && days < LONG_TERM) {
   return price -= MIDDLE_TERM_DISCOUNT;
}

if (days >= LONG_TERM) {
   return price -= LONG_TERM_DISCOUNT;
}
```
without `return price` in the end